.github/workflows/build.yml

```diff
@@ -118,6 +117,9 @@ jobs:
         with:
           images: ghcr.io/espressosystems/cape/wallet
 
+      - name: Generate base image
+        run: nix-shell --run build-docker-base
```

Comment:
This is a bit wasteful and we don't really need to activate the nix shell but I'm not the `nix build ...` command works without it.

Reply:
Will give it a go when I get around to it.

Reply:
This is done. The docker workflow just takes seconds now.